server/server.js

```diff
@@ -1,25 +1,110 @@
 const express = require('express');
+
 const path = require('path');
+
 const app = express();
-const port = 8080;
+const port = 8070;
```

Comment:
```suggestion
const port = 8080;
```

No need to change this now